File: sem-1/acctosquence/12.c
// find the smallest of 3 number

#include <stdio.h>

void main() {
    float a, b, c;

    printf("Enter value of A: ");
    scanf("%f", &a);

    printf("Enter value of B: ");
    scanf("%f", &b);

    printf("Enter value of C: ");
    scanf("%f", &c);

    if ((a < b) && (a < c)) {
        printf("\nA is the smallest\n");

    } else if ((b < c) && (b < a)) {
        printf("\nB is the smallest\n");

    } else if ((c < b) && (c < a)) {
        printf("\nC is the smallest\n");

    } else {
        printf("invalid\n");
    }
}

File: sem-2/ITC101/assign/12.py
list_num = [31,441,5,1,53,215,1,42,14,31,2]


s = l = list_num[0]

for i in list_num:
    if i > l:
        l = i

    if i < s:
        s = i

print(f"Smallest number in list: {s}")
print(f"Largest number in list: {l}")